pkg/internal/checkup/trex/configfiles_test.go

```diff
@@ -134,15 +135,15 @@ func TestExecutionScript(t *testing.T) {
 func TestSystemdUnitFile(t *testing.T) {
 	actualSystemdUnitFile := trex.GenerateSystemdUnitFile()
 
-	expectedSystemdUnitFile := `[Unit]
+	expectedSystemdUnitFile := fmt.Sprintf(`[Unit]
```

Comment:
Please consider using a string builder instead of a long format string.
IMO its more readable.

Reply:
`GenerateSystemdUnitFile` is using a string builder.
As this Unit test checks the final output of the string, I prefer using the full string as I expect it to come out of `GenerateSystemdUnitFile`

Reply:
> GenerateSystemdUnitFile is using a string builder.

This is an implementation detail that IMO does not affect the test.

The advantage of using a string builder is that its easier to relate a placeholder with its value.
I won't insist on this change.

Reply:
I disagree. If we're setting `expectedSystemdUnitFile` to be a copy paste of the function creating it - we are not actually checking it. In this example it's fairly simple but in general I prefer using the final output of the string I expect to see.